README.md

```diff
@@ -260,13 +272,22 @@ spark-submit \
   --job-name "demo_photon_ml_logistic_regression"
 ```
 
-There is also a more complex script demonstrating advanced options and customizations of using Photon ML at  [example/run_photon_ml.driver.sh](https://github.com/linkedin/photon-ml/blob/master/examples/run_photon_ml_driver.sh).
+There is also a more complex script demonstrating advanced options and customizations of using Photon ML at [example/run_photon_ml.driver.sh](https://github.com/linkedin/photon-ml/blob/master/examples/run_photon_ml_driver.sh).
 
 Detailed usages are described via command:
 ```bash
 ./run_photon_ml.driver.sh [-h|--help]
 ```
 **Note**: not all configurations are currently exposed as options in the current script, please directly modify the confs if any customization is needed.
 
+#### Running Photon ML on Other Cluster Mode
+Similar to running Photon ML or other general Spark applications on YARN, running Photon ML on other cluster mode (e.g.
```

Comment:
"Similarly to running Photon ML (or other general Spark applications) on YARN, running Photon ML on other cluster modes... "